webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -16,8 +17,8 @@ export function makeGetScheduledPostsByTeam(): (state: GlobalState, teamId: stri
         'makeGetScheduledPostsByTeam',
         (state: GlobalState) => state,
         (state: GlobalState, teamId: string, includeDirectChannels: boolean) => includeDirectChannels,
-        (state: GlobalState, teamId: string) => state.entities.scheduledPosts.byTeamId[teamId],
-        (state: GlobalState) => state.entities.scheduledPosts.byTeamId.directChannels,
+        (state: GlobalState, teamId: string) => state.entities.scheduledPosts.byTeamId[teamId] || [],
+        (state: GlobalState) => state.entities.scheduledPosts.byTeamId.directChannels || [],
```

Comment:
I hope this doens't happen often, but the way this is implemented, if `byTeamId` or `directChannels` are not defined, the selector becomes useless, because it will run and create a new scheduledPosts list on every call.

Creating a constant for the empty list should be enough.

Reply:
Done.